herd/tests/instructions/AArch64/A25.litmus

```diff
@@ -0,0 +1,11 @@
+AArch64 A25
+
+(* Tests ldrsh instruction *)
+{ 0:X0 = 1; 0:X1 = x; int16_t x = 65535; }
```

Comment:
Same suggestions as for test `A24`: either declare `0:X0` of type `int64_t` or use `W` registers in code.